src/celo/chainConfig.ts

```diff
@@ -4,6 +4,7 @@ import { formatters } from './formatters.js'
 import { serializers } from './serializers.js'
 
 export const chainConfig = {
+  blockTime: 2_000,
```

Comment:
```suggestion
  blockTime: 1_000,
```

celo's block time is 1 second  https://specs.celo.org/deployments.html?highlight=second#mainnet